website/docs/cli/install/apt.mdx

```diff
@@ -91,6 +91,8 @@ following distribution releases:
 * Ubuntu 20.10 (`groovy`)
 * Ubuntu 21.04 (`hirsute`)
 * Ubuntu 21.10 (`impish`)
+* Ubuntu 22.04 (`jammy`)
+* Ubuntu 22.10 (`kinetic`)
 
 No repositories are available for other Debian or Ubuntu versions or
 any other APT-based Linux distributions. If you add the repository using
```

Comment:
Since this list is not being updated in a timely fashion, I think this paragraph should be updated to reflect this tardiness.